recipes/r-fedmatch/meta.yaml

```diff
@@ -0,0 +1,106 @@
+{% set version = '2.0.6' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-fedmatch
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/fedmatch_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/fedmatch/fedmatch_{{ version }}.tar.gz
+  sha256: e2e143c3cdda215920f76abf1e1528e2ec000bbe3bb52a4d51559989c3909419
+
+build:
+  merge_build_host: True  # [win]
+  missing_dso_whitelist:
+    - '*/R.dll'  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ stdlib("c") }}
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - llvm-openmp                  # [osx]
+    - libgomp                      # [linux]
+  host:
+    - r-base
+    - r-bh
+    - r-rcpp
+    - r-snowballc
+    - r-data.table
+    - r-forcats
+    - r-magrittr
+    - r-purrr
+    - r-scales
+    - r-stringdist
+    - r-stringr
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-bh
```

Comment:
Boost headers are only needed at build time. We usually remove it from `run:`, when only in `LinkingTo`, i.e.,

```suggestion
```